microsoft-dynamics-365/2.5/modules/ROOT/pages/microsoft-dynamics-365-connector-studio.adoc

```diff
@@ -0,0 +1,212 @@
+= Use Anypoint Studio to Configure Microsoft Dynamics 365 Connector 2.4  - Mule 4
+:page-aliases: connectors::microsoft/microsoft-dynamics-365-connector-studio.adoc
+:page-aliases: connectors::microsoft/microsoft-dynamics-365-connector-design-center.adoc
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure a source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+
+When you run the connector, you can view the app log to check for problems, as described in <<view-app-log,View the App Log>>.
+
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:microsoft-dynamics-365-connector-reference.adoc[Microsoft Dynamics Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Microsoft Dynamics 365 Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `dynamics 365` in the search field.
+. Click *Microsoft Dynamics 365* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+
+[[configure-input-source]]
+== Configure a Source
```

Comment:
In this release, we will add support for On Deleted Object source also. I think it should be described here as well?